stores/sql/mysql/migrations/main/schema.sql

```diff
@@ -225,9 +225,11 @@ CREATE TABLE `ephemeral_accounts` (
   `balance` longtext,
   `drift` longtext,
   `requires_sync` tinyint(1) DEFAULT NULL,
+  `owner` varchar(128) NOT NULL,
```

Comment:
`worker.ID` needs a length check if we're capping this at 128 (which I believe is done silently but not sure)

Reply:
By default MySQL checks the length. Verified this on my db.

Reply:
Ok, for completeness though, SQLite will not truncate but will persist a value here that's longer than 128 characters without error.

Reply:
Yes, that's correct. SQLite is a bit more forgiving there. It also won't complain about the index when not specifying a length.